openapi/index.yaml

```diff
@@ -610,7 +610,7 @@ definitions:
     required:
       - email
   EmailAddress:
-    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v22.5.0/openapi/definitions.yaml#/EmailAddress"
+    $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/a517079948bbb50f16cc5267f4fe880442d12642/openapi/definitions.yaml#/EmailAddress"
```

Comment:
why commit hash instead of tag version? is there a new guideline I mistakenly missed?